src/components/migrationToV3.stories.mdx

```diff
@@ -27,7 +27,7 @@ Amazon Chime SDK React Components Library v3 includes major improvements for com
 - **WebRTC metrics**: Publish the standardized WebRTC metrics for all supported browsers. Deprecate `useBandwidthMetrics` hook in favor of `useMediaStreamMetrics` hook.
 - **Logger**: Add `LoggerProvider` and `useLogger` components to enable universal logging in component library.
 
-Check out our <a target="_blank" href="https://github.com/aws/amazon-chime-sdk-component-library-react/blob/2e9ae19fb6ea72a6611a7d7188744c1db45d8e09/CHANGELOG.md#L20">CHANGELOG for 3.0.0</a> for more details.
+Check out our [CHANGELOG for 3.0.0](https://github.com/aws/amazon-chime-sdk-component-library-react/blob/2e9ae19fb6ea72a6611a7d7188744c1db45d8e09/CHANGELOG.md#L20) for more details.
```

Comment:
Do we need to update other links in this PR?

Reply:
Nope, I thought you had a script which would easily update this 😅 thats why for test I made this change to know it works :). Happy to put another PR updating rest.

Reply:
Oh got it, let me raise a PR to update those, and I found several typos need to be fixed.